components/brave_new_tab_ui/containers/newTab/settings.tsx

```diff
@@ -55,6 +55,7 @@ export interface Props {
   setMostVisitedSettings: (show: boolean, customize: boolean) => void
   toggleShowRewards: () => void
   toggleShowBraveTalk: () => void
+  toggleShowBraveVPN: () => void
```

Comment:
if you can get `usePref` working for you you won't need to modify this, as its only drilling props

Reply:
Deleted.